en/docs/install-and-setup/setup/distributed-deployment/configuring-wso2-identity-server-as-a-key-manager.md

```diff
@@ -250,6 +250,8 @@ Follow the instructions below to set up and configure the databases for the WSO2
     ```
 7. If you wish to encrypt the OAuth2 Keys (access tokens, client secrets, and authorization codes), follow the steps given in [Encrypting OAuth Keys](https://is.docs.wso2.com/en/5.10.0/learn/testing-oidc-encrypted-id-token-with-is/#enable-id-token-encryption), which is in the WSO2 Identity Server 5.10.0 documentation, and apply the relevant configurations in the `<IS_HOME>/repository/conf/deployment.toml` file to enable the feature.
 
+8. If you are setting up the deployment in a local environment (non production) and wish to use the existing keystore and truststore (with self signed certificate) shipped by default with the product distributions, replace the keystore in `<IS_HOME>/repository/resources/security/wso2carbon.jks` and trustore in `<IS_HOME>/repository/resources/security/client-truststore.jks` with the ``<APIM_HOME>/repository/resources/security/wso2carbon.jks` and  `<APIM_HOME>/repository/resources/security/client-truststore.jks` respectively. Make sure to follow this step before starting either of the servers for the first time.
```

Comment:
"If you are using the existing keystore and truststore (with self signed certificate) shipped by default with the product distributions, replace the keystore in ..."

(Please note that in a production environment, it is not recommended to use the default keystores. Instead, it is recommended to [create new keystores](https://apim.docs.wso2.com/en/4.1.0/install-and-setup/setup/security/configuring-keystores/keystore-basics/creating-new-keystores) with new keys and certificates.)

Reply:
Shall we add as above?
"local environment (non production) **and** wish to use " : two conditions makes the statement complex.